.ci/packaging.groovy

```diff
@@ -188,9 +188,15 @@ def tagAndPush(name){
   if("${env.SNAPSHOT}" == "true"){
     libbetaVer += "-SNAPSHOT"
   }
+
+  def imageName = "${name}"
+  if (env.CHANGE_ID?.trim()) {
+    imageName = "${name}/pr-${env.CHANGE_ID}"
```

Comment:
the image will have a mane like `docker.elastic.co/observability-ci/filebeat/pr-1000:a2b3a2b3bb12b2b1212b2` for me is weird, I'd like to release `docker.elastic.co/observability-ci/filebeat:a2b3a2b3bb12b2b1212b2` and `docker.elastic.co/observability-ci/filebeat:pr-1000`

Reply:
Agreed. I sent a commit with this change, as I see it more convenient too